CHANGELOG.md

```diff
@@ -12,6 +12,28 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.25.0] - 2024-12-31
+
+### Release highlights
+
+It's the holidays, and this release was overall pretty quiet, without many major
+changes. Two select improvements:
+
+* Improvements to configuration management, including support for [conditional
+  variables](docs/config.md#conditional-variables) in config files.
+
+* Large files will no longer cause snapshots (and commands like `jj st`) to
```

Comment:
nit: since basically all commands do snapshotting, maybe just "no longer cause commands to fail"